supertokens_python/recipe/session/session_class.py

```diff
@@ -50,6 +52,12 @@ class Session(SessionContainer):
     async def attach_to_request_response(
         self, request: BaseRequest, transfer_method: TokenTransferMethod
     ) -> None:
+        if self.req_res_info is None:
```

Comment:
Why are we branching here? Couldn't we just overwrite the entire prop? (My Python knowledge may be lacking)

Reply:
Good point. Done.